ouroboros-consensus/src/unstable-consensus-testlib/Test/Util/TestBlock.hs

```diff
@@ -823,3 +825,66 @@ instance Serialise ptype => HasBinaryBlockInfo (TestBlockWith ptype) where
       }
 
 instance (Serialise ptype, PayloadSemantics ptype) => SerialiseDiskConstraints (TestBlockWith ptype)
+
+-----
+
+-- | Given a point to a chain of length L, generates a 'SwitchFork' that
+-- switches to the "next" block of length L, where "next" is determined by
```

Comment:
Should this be `chain of lenght L`?

Reply:
That's a reasonable expectation! But because a block chain has the Merkle tree property, a chain and the block at its tip are in many ways indistinguishable. In particular, the length of a block is exactly the length of _the_ chain that ends in that block.

So you'll see this language elsewhere; sometimes "chain" or "block" feels more appropriate depending on context.